jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -207,8 +201,6 @@ ecma_builtin_get (ecma_builtin_id_t builtin_id) /**< id of built-in to check on
     ecma_instantiate_builtin (builtin_id);
   }
 
-  ecma_ref_object (JERRY_CONTEXT (ecma_builtin_objects)[builtin_id]);
```

Comment:
We should add a note to the method's description that the object's ref count is not increased just like in the case of the `ecma_builtin_get_global` call.

Reply:
Good catch!